FHEM/00_SIGNALduino.pm

```diff
@@ -2357,18 +2385,25 @@ sub SIGNALduino_Parse_MU($$$$@)
 							Debug "$name: padding 0 bit to bit_msg array" if ($debug);
 						}
 			
-						SIGNALduino_Log3 $name, 5, "$name: dispatching bits: @bit_msg";
-						my $dmsg = SIGNALduino_b2h(join "", @bit_msg);
+						my $dmsg = join ("", @bit_msg);
+						SIGNALduino_Log3 $name, 5, "$name: dispatching bits: $dmsg";
+						if (index($dmsg, "F") == -1 ) {
+							$dmsg = SIGNALduino_b2h($dmsg);
+						}
```

Comment:
Damit wird nicht nach hex gewandelt, wenn wenn float enthalten ist.
Universeller ist es wahrscheinlich, wenn ich dies mit einem neuen Eintrag in der Protocolllist mache.
Z.B. dispatchBin
und dann die Abfrage
```
if (defined($ProtocolListSIGNALduino{$id}{dispatchBin})) {
	my $dmsg = join ("", @bit_msg);
} else {
	while (scalar @bit_msg % $padwith > 0)  ## will pad up full nibbles per default or full byte if specified in protocol
	{
		push(@bit_msg,'0');
		Debug "$name: padding 0 bit to bit_msg array" if ($debug);
	}
	my $dmsg = join ("", @bit_msg);
	$dmsg = SIGNALduino_b2h($dmsg);
}
```